docs/getting-started/integrating-codacy-with-visual-studio-code.md

```diff
@@ -19,46 +79,29 @@ Use this extension to get the full list of problems found by Codacy for a pull r
 
 ## Getting pull request quality and coverage data
 
-To see Codacy quality and coverage data for the pull request you're working on, follow these steps:
+To see Codacy quality and coverage data for an open pull request, follow these steps:
 
 1.  Open the repository directory in VS Code.
 
     !!! note
         If the repository isn't on Codacy yet, [add it to Codacy](../organizations/managing-repositories.md#adding-a-repository) first.
 
-1.  Check out the branch associated with the open pull request you're working on.
-
 1.  Open the main view by clicking the **Codacy logo** in the activity bar or the **Codacy tab** in the status bar.
```

Comment:
If we refer to the **Codacy tab** in the status bar, we should include it on the screenshot. 🤔

Reply:
Done!